src/pudl/package_data/eia860m/column_maps/generator_existing.csv

```diff
@@ -12,8 +12,8 @@ operational_status_code,status,status,status,status,status,status,status,status,
+generator_operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year,operating_year
```

Comment:
@aesharpe / @e-belfer  i noticed during my changelog 860m building that the column names in this mapping was resulting in the 860m operating date probably not getting integrated because the column name was inconsistent with the rest of the 860 in the column maps.